database/seeds/VoteUniqueViewSeeder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class VoteUniqueViewSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $arrayOfVoteUniqueViews= [];
+        for ($i=0;$i<500;$i++){
+            $arrayOfVoteUniqueViews[] = [
+                'user_id' => \App\Models\User::all()->random()->id,
```

Comment:
Will be better do declare classes from other namespaces (for ex. \App\Models\User) at the top of file.